server/test/unit/server/db/test_connection.py

```diff
@@ -1,18 +1,23 @@
-import unittest
+import unittest, unittest2
```

Comment:
last comment, I swear. ;) ```unittest2``` is a third-party module, so it should be in the second group of imports.